modules/reference/pages/rpk/rpk-topic/rpk-topic-add-partitions.adoc

```diff
@@ -2,6 +2,8 @@
 
 Add partitions to existing topics.
 
+NOTE: Existing data is not redistributed to the newly added partitions. This only occurs as new data is added.
```

Comment:
```suggestion
NOTE: Existing topic data is not redistributed to the newly-added partitions. This only occurs as new data is added.
```

Reply:
I'm a little unclear/unsure what "This" at the beginning of the second sentence refers to...Here's a brief bit of guidance on that: https://www.quora.com/Are-there-any-rules-about-writing-this-before-starting-a-sentence-or-using-that-If-so-what-are-they

Reply:
Is it the following?
 Existing topic data is distributed to newly-added partitions only after running this command.